aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -598,6 +613,31 @@ class QConv1dPackWeightInt8 final {
     padding = quant_utils::MakeArgForConv1d(padding, 0);
     output_padding = quant_utils::MakeArgForConv1d(output_padding, 0);
     dilation = quant_utils::MakeArgForConv1d(dilation, 1);
+
+#ifdef USE_FBGEMM
+  if (ctx.qEngine() == at::QEngine::X86) {
+#if AT_MKLDNN_ENABLED()
+    bool no_vnni = !cpuinfo_has_x86_avx512vnni() && !cpuinfo_has_x86_avx512_4vnniw();
+    bool w_sym_quant =
+        onednn_utils::is_weight_symmetric_quant(weight, transpose);
+    bool prefer_fbgemm = no_vnni || (groups > 100) || !w_sym_quant;
+    if (prefer_fbgemm) {
```

Comment:
What's the reason not calling `onednn_utils::should_use_onednn_quant` here?

Reply:
Well, I think it's a bug. I will fix it.